c#/Mandoline.Api.Examples/Tests/Tests.csproj

```diff
@@ -76,35 +26,22 @@
     </Otherwise>
   </Choose>
   <ItemGroup>
-    <Compile Include="DownloadShapedTests.cs" />
-    <Compile Include="DownloadTests.cs" />
-    <Compile Include="InfoTests.cs" />
-    <Compile Include="SelectionTest.cs" />
-    <Compile Include="TestOutput.cs" />
-    <Compile Include="UserTests.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
-  </ItemGroup>
-  <ItemGroup>
-    <ProjectReference Include="..\Core\Core.csproj">
-      <Project>{1eba8b7b-4937-4895-94c4-ba99c0b716c7}</Project>
-      <Name>Core</Name>
-    </ProjectReference>
-  </ItemGroup>
-  <ItemGroup>
-    <None Include="App.config" />
-    <None Include="AppSettings.config">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </None>
-    <None Include="packages.config" />
+    <ProjectReference Include="..\Core\Core.csproj" />
   </ItemGroup>
   <ItemGroup>
     <Content Include="Docs\curl_examples.txt" />
     <Content Include="Docs\Readme.txt" />
   </ItemGroup>
   <ItemGroup>
-    <Analyzer Include="..\packages\StyleCop.Analyzers.1.0.0\analyzers\dotnet\cs\Newtonsoft.Json.dll" />
-    <Analyzer Include="..\packages\StyleCop.Analyzers.1.0.0\analyzers\dotnet\cs\StyleCop.Analyzers.CodeFixes.dll" />
-    <Analyzer Include="..\packages\StyleCop.Analyzers.1.0.0\analyzers\dotnet\cs\StyleCop.Analyzers.dll" />
+    <PackageReference Include="log4net" Version="2.0.12" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.5.0-preview-20221003-04" />
```

Comment:
Hmm, is there a stable version of the Microsoft.NET.Test.Sdk that we can use instead of a preview version? We should avoid preview versions as they are open to change and potential bugs.